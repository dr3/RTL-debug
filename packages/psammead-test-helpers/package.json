{
  "name": "@bbc/psammead-test-helpers",
  "version": "0.2.0",
  "main": "dist/index.js",
  "description": "A collection of helper methods for react snapshot testing with support for styled components",
  "dependencies": {
    "jest-styled-components": "^6.3.1",
    "react-testing-library": "^5.3.1"
  }
}
