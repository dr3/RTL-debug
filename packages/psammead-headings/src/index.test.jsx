import React from 'react';
import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
import Headline from './index';

describe('Headline component', () => {
  shouldMatchSnapshot(
    'should render correctly',
    <Headline>This is my headline.</Headline>,
  );
});
