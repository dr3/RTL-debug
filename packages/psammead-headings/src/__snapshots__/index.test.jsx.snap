// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline component should render correctly 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdVaJa rOCEJ"
  >
    This is my headline.
  </h1>
</DocumentFragment>
`;
